resources/assets/css/_document.css

```diff
@@ -371,3 +371,13 @@ main .documentation-content .heading-permalink:hover {
 .documentation-content dl dd {
     @apply pb-6;
 }
+
+/* purgecss start ignore */
+.twitter-embed-wrapper {
+    @apply dark:hidden flex flex-col items-center;
```

Comment:
Note: I'm using flex-col because at some point, while the twitter script did not replace the twit with the iframe yet, we have two HTML child items that otherwise are shown in rows.